src/HelixServer.js

```diff
@@ -63,7 +63,7 @@ function executeTemplate(ctx) {
     owner: ctx.config.contentRepo.owner,
     repo: ctx.config.contentRepo.repo,
     ref: ctx.config.contentRepo.ref,
-    path: `${ctx.resourcePath.substr(1)}.md`, // either pipeline or git-server don't like the double slash here
+    path: `/${ctx.resourcePath.split('/').filter(e => e !== '').join('/')}.md`,
```

Comment:
this is certainly the wrong place to do this.
`ctx.resourcePath` should already be sanitized.

just use `${ctx.resourcePath}.md` as before.

btw: I think the `.md` is conceptually not correct here. I would argue that the `path` is (similar to sling) is w/o the extension. but this is another discussion.

Reply:
I agree. The `resourcePath` should be set (if at all) by the thing that is responsible for fetching the resource, i.e. the pipeline. In addition to having non-md resources, you might also have non-GitHub resources.

Reply:
The `resourcePath` is computed here https://github.com/adobe/petridish/blob/master/src/RequestContext.js#L28-L47
I don't see how you can have multiple leading slashes.
The patch here is only for Petridish, if the multiple leading slashes cases you observe occurs on "prod" we should rather investigate there and fixed.
I just added the sanitise here as a safety and to align with the hack which was already in place but as I mentioned before, I never saw it happen (on Petridish).
I can remove it for now.